deps/rabbit/src/rabbit_file.erl

```diff
@@ -244,6 +244,7 @@ recursive_delete1(Path) ->
         false -> case prim_file:delete(Path) of
                      ok              -> ok;
                      {error, enoent} -> ok; %% Path doesn't exist anyway
+                     {error, ebusy}  -> ok; %% Ignore (rabbitmq/rabbitmq-server#11047)
```

Comment:
Please say why instead of linking to an issue that won't exist in 10 years. :)

"Directory is in use by the kernel; can't be deleted"